profiles/common/modules/custom/multisite_cookie_consent/multisite_cookie_consent.module

```diff
@@ -50,8 +50,7 @@ function multisite_cookie_consent_init() {
   if (current_path() != 'cookie-notice') {
     // Notice url.
     $notice_page = url('cookie-notice', array('absolute' => TRUE));
-    // Add for example external.
-    drupal_add_js('http://ec.europa.eu/wel/like/js/likedislike.js', 'external');
```

Comment:
We cannot remove this line, we should change the link.
for example remplace by http://europa.eu/webtools/load.js

Reply:
Why do you want to replace this line? This feature no longer exists, so the js is no longer needed.

Reply:
Module was removed from the platform